DeviceComunication/DeviceComunication_CZ.md

```diff
@@ -1048,3 +1092,9 @@ Zařízení vypočítá počet měření, které je potřeba udělat před odesl
 
 Zařízení pak bude čekat mezi měřeními 4 minuty a po každých dvou měření odešle zprávu. Measure
 zprávy se tedy budou odesílat každých 8 minut.
+
+## Zjednodušená implementace potvrzování
+Protože zařízení ve výchozím stavu vyžaduje potvrzení některých zpráv 
+(určuje [5.byte v hlavičce](#5byte---nb-iot)), je nutné implementovat alespoň 
+základní komunikaci ze serveru do zařízení. Na tyto zprávy je možné zjednodušeně 
+odpovídat staticky payloadem `0000000001FF00` který zajistí potvrzení přijaté zprávy.
```

Comment:
Jeste je teda potřeba zkontrolovat ten algoritmus pro posilani downlinku na Lora jestli bude fungovat pokud vypadnou ty zpravy.

Reply:
Algoritmus pro odeslání (označování zpráv) sedí. Drobně jsem upravil texty kolem toho týkající se potvrzování aby to bylo jasnější, u schémátka výpadku zprávy při joinu je připsané že je to i případ vypadení zprávy.